src/ui-test/tests/camel.settings.test.ts

```diff
@@ -80,10 +84,53 @@ describe('Camel User Settings', function () {
 
     });
 
+    describe('Update JBang Version', function () {
+
+        const customJBangVersion = '3.20.5';
+
+        beforeEach(async function () {
+            await (await new ActivityBar().getViewControl('Explorer')).openView();
```

Comment:
@djelinek I think it can be just moved from separate `describe` to one `describe`

from:

```
 describe('Update JBang Version', function () {...}
    before(async function () {...}
    after(async function () {...}
    describe('Update Camel Version', function () {...}
       before(async function () {...}
       after(async function () {...}
       it(`Should use '${customCamelVersion}' user defined Camel version`, async function () {...}
    describe('Update JBang Version', function () {...}
       beforeEach(async function () {...}
       afterEach(async function () {...}
       it(`Should use default JBang version`, async function () {...}
       it(`Should use user defined JBang version '${customJBangVersion}'`, async function () {...}
```

to

```
 describe('Update JBang Version', function () {...}
    before(async function () {...}
    after(async function () {...}
    beforeEach(async function () {...}
    afterEach(async function () {...}
    it(`Should use '${customCamelVersion}' user defined Camel version`, async function () {...}
    it(`Should use default JBang version`, async function () {...}
    it(`Should use user defined JBang version '${customJBangVersion}'`, async function () {...}
```

WDYT? It will eliminate redundancies and no need to make extra functions

Reply:
it depends.. personally I like the splitting into more inner describes which brings some categorisation but both will work of course

Reply:
I mean I would like to still have some order and space between test scenarios if there is more inside one file. Not to have one describe and many of tests.. it could became easily a bit messy

use more inner describes should allow us to have more tests from same category in one place but still somehow logically divided